### PR TITLE
Convert Trac Wiki markup to Markdown

### PR DESCRIPTION
**Pull request by aaaaalbert**
Also add a few bits of formatting where it makes sense.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1009,19 +1009,22 @@ perform or avoid the evaluation of the askokcancel function.
...
 
-  Call subprocess.Popen with a first argument that looks like ['command',
-  'arg1', 'arg2', ...].   Do not use 'command arg1 arg2 ...'!
+  Call subprocess.Popen with a first argument that looks like `['command',
+  'arg1', 'arg2', ...]`.   Do not use `'command arg1 arg2 ...'`!
 
 
-### Avoid changing directory (#487) ###
+### Avoid changing directory ###
+(See SeattleTestbed/attic#487)
```
Comment (vladimir-v-diaz) [inline README.md]: Should this be a link?  `SeatleTestbed` is a different GitHub organization (this one's `secure-systems-lab`).

File: README.md [hunk 2]
```diff
@@ -1009,19 +1009,22 @@ perform or avoid the evaluation of the askokcancel function.
...
 
   Changing the current directory can cause multithreaded programs to break is
   scary ways and is usually an indication of bad programming style.   Don't do
   it!
 
-### Don't use mutable objects as argument defaults (#828) ###
+### Don't use mutable objects as argument defaults ###
+(See SeattleTestbed/attic#828)
```
Comment (vladimir-v-diaz) [inline README.md]: Same here.  Should we link to #828?

File: README.md [hunk 3]
```diff
@@ -637,25 +637,25 @@ example that uses globals in a good way is a program that keeps a cache.
 Function names should be lowercase, with words separated by underscores as
 necessary to improve readability.
 
-!mixedCase is allowed only in contexts where that's already the prevailing
+mixedCase is allowed only in contexts where that's already the prevailing
 style (e.g. threading.py), to retain backwards compatibility.
```
Comment (lukpueh) [inline README.md]: *'threading.py'* as example for mixedCase? :)  Maybe we can give a hint saying that this is the context and not the example.

File: README.md (hunk 1 above)
Comment (aaaaalbert) [inline README.md]: Addressed in d41067dc72e11d3a49672c79e27ac9866020fc44.

File: README.md (hunk 2 above)
Comment (aaaaalbert) [inline README.md]: Yes, you are right! Addressed in d41067dc72e11d3a49672c79e27ac9866020fc44.

File: README.md (hunk 3 above)
Comment (aaaaalbert) [inline README.md]: Valid point. Addressed in d41067dc72e11d3a49672c79e27ac9866020fc44.

**Comment by aaaaalbert** [comment]
Thanks for embracing the PR :smile:. I think I've addressed your comments, so please take another look and proceed as you deem suitable.

**Pull request merged**